### PR TITLE
Add janitor/control plane services, cleanup.

### PR DESCRIPTION
**Pull request by fmassot**
```
k get svc

NAME                                  TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                         AGE
qw-gharchive-quickwit-control-plane   NodePort    172.20.146.158   <none>        7280:32159/TCP,7281:30186/TCP   6m56s
qw-gharchive-quickwit-headless        ClusterIP   None             <none>        7282/UDP                        6m56s
qw-gharchive-quickwit-indexer         NodePort    172.20.225.155   <none>        7280:31993/TCP,7281:32264/TCP   6m56s
qw-gharchive-quickwit-janitor         NodePort    172.20.7.94      <none>        7280:30004/TCP,7281:31495/TCP   6m56s
qw-gharchive-quickwit-metastore       NodePort    172.20.138.136   <none>        7280:31656/TCP,7281:31392/TCP   6m56s
qw-gharchive-quickwit-searcher        NodePort    172.20.126.208   <none>        7280:32179/TCP,7281:32017/TCP   6m56s

```

### CODE REVIEW

File: charts/quickwit/templates/control-plane-deployment.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
     matchLabels:
       {{- include "quickwit.control_plane.selectorLabels" . | nindent 6 }}
   strategy:
-    {{- toYaml .Values.control_plane.updateStrategy | nindent 4 }}      
+    type: Recreate
```
Comment (fmassot) [inline charts/quickwit/templates/control-plane-deployment.yaml:13]: Ensures only one control plane in the metastore when upgrading the chart.

File: charts/quickwit/values.yaml [hunk 2]
```diff
@@ -155,11 +155,6 @@ metastore:
     #   cpu: 100m
     #   memory: 128Mi
 
-  ## Pod distruption budget
-  podDisruptionBudget: {}
```
Comment (fmassot) [inline charts/quickwit/values.yaml:159]: cleanup: was not used in the chart.

File: charts/quickwit/values.yaml [hunk 3]
```diff
@@ -187,8 +182,6 @@ metastore:
   affinity: {}
 
 control_plane:
-  replicaCount: 1
```
Comment (fmassot) [inline charts/quickwit/values.yaml:190]: we don't want more than 1 control plane

File: charts/quickwit/values.yaml [hunk 4]
```diff
@@ -248,14 +233,6 @@ janitor:
     #   cpu: 100m
     #   memory: 128Mi
 
-  ## Pod distruption budget
```
Comment (fmassot) [inline charts/quickwit/values.yaml:251]: was not used

**Review by guilload — APPROVED** [review]

**Pull request merged**